.github/workflows/main.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     with:
       CONTEXT: dev
       ROLE_TO_ASSUME:
-        arn:aws:iam::631692904429:role/sagebase-github-oidc-sage-ProviderRoles3filehostin-8LPZFSITXB0I
+        arn:aws:iam::631692904429:role/sagebase-github-oidc-sage-ProviderRole
```

Comment:
I don't think this ARN is correct, if the role name includes the stack name that created it, then wouldn't this role include `sage-bionetworks-s3-file-hosting-infra`?

Reply:
fixed